build/lib.linux-x86_64-2.7/sematch/semantic/similarity.py

```diff
@@ -165,7 +165,7 @@ def jcn(self, c1, c2):
         lcs_ic = 2.0 * lcs_ic
         if c1_ic == 0.0 or c2_ic == 0.0:
             return 0.0
-        return 1.0 / 1 + (c1_ic + c2_ic - lcs_ic)
+        return 1.0 / (1 + c1_ic + c2_ic - lcs_ic)
```

Comment:
`c1_ic + c2_ic - lcs_ic` is the jiang-corinth distance AFAICT. So the question here is what is the formula to convert this distance to a similarity?